javaslang-test/src/main/java/javaslang/test/Arbitrary.java

```diff
@@ -117,7 +163,30 @@
         Objects.requireNonNull(f, "f is null");
         return f.apply(this);
     }
-    
+
+    /**
+     * Generates an arbitrary value from a fixed set of values
+     *
+     * @param values A fixed set of values
+     * @param <U> Type of generator value
+     * @return A new generator
+     */
+    @SafeVarargs
+    @SuppressWarnings("varargs")
+    static <U> Arbitrary<U> fixed(U... values) {
+        return forAll(Gen.choose(values));
+    }
+
+    /**
+     * Generates an arbitrary value from a given generator
+     * @param generator A generator to produce arbitrary values
+     * @param <U> Type of generator value
+     * @return A new generator
+     */
+    static <U> Arbitrary<U> forAll(Gen<U> generator) {
+      return size -> generator;
+    }
```

Comment:
Hi @talios, finally I'm able to answer... Thanks - the code looks great.
Instead of `fixed` and `forAll` I would name the methods `of(T...)` and `ofAll(Gen)` to follow the naming scheme of the collections. What do you think?
We already have `gen.arbitrary()` but I think having `Arbitrary.ofAll(Gen)` is an intuitive place to look for a factory.